README.md

```diff
@@ -184,14 +230,44 @@ Arrow.Model.extend('authorWithArticles', {
 	}
 })
 ```
+
+#### Joining with Single Field from Multiple Children
+
+It's also possible that we don't want every whole post object returned, but to save bandwidth, only the title of each post is necesarry. This is also possible! We just need to specify the name of the field we want to use, but this time we set the field type as Array and add a parameter "multiple" to the join metadata.
```

Comment:
"This is also possible!"  ... contain your excitement :)

Give example models data and output coz this makes little sense unless you see it. Actually that would be a good suggestion for them all.